css/build.css

```diff
@@ -120,187 +121,218 @@
 }
 /* METRO SIZES */
 .panels.size11 {
-  width: 145px;
+  width: 50%;
   height: 145px;
 }
 
 .panels.size21 {
-  width: 300px;
+  width: 100%;
   height: 145px;
 }
 
 .panels.size22 {
-  width: 300px;
+  width: 100%;
   height: 300px;
 }
 @media (min-width: 420px) {
   .panels.size11 {
-    width: 160px;
+    width: 33.3%;
     height: 160px;
   }
 
   .panels.size21 {
-    width: 330px;
+    width: 50%;
     height: 160px;
   }
 
   .panels.size22 {
-    width: 330px;
+    width: 50%;
     height: 330px;
   }
 }
+@media (min-width: 600px) {
```

Comment:
@YaroslavOvdii Why was the 600px suddenly added in the latest version of the fix?

Reply:
@tonytlwu That is because in range from 600px to 640px, freewall can fit only one medium size item, but from 640 and above it could fit two medium size item. That is why I've added 600px so existing users wont affect it. And 0nly with 600px(size of the preview in the studio) we could replicate this bug so it was specified here as well to avoid any bugs in the future.